src/playlist-loader.js

```diff
@@ -241,10 +241,18 @@ const PlaylistLoader = function(srcUrl, hls, withCredentials) {
   loader.isLowestEnabledRendition_ = function() {
     let media = loader.media();
 
-    if (!media || !media.attributes) {
+    if (!media) {
       return false;
     }
 
+    if (!media.attributes) {
```

Comment:
I think a better way would be to just always return `true` if there is only a single variant.